File: livingrimoire start here/LivinGrimoire java/src/Auxiliary_Modules/Differ.java
package Auxiliary_Modules;

public class Differ {
    private int powerLevel = 90;
    private int difference = 0;

    public int getPowerLevel() {
        return powerLevel;
    }

    public int getDifference() {
        return difference;
    }
    public void clearPowerLVDifference(){
        difference = 0;
    }
    public void samplePowerLV(int pl){
        // pl is the current power level
        difference = pl - powerLevel;
        powerLevel = pl;
    }
}

File: livingrimoire start here/LivinGrimoire Kotlin/kotlin/auxiliary_modules/Differ.kt
package auxiliary_modules

class Differ {
    var powerLevel = 90
        private set
    var difference = 0
        private set

    fun clearPowerLVDifference() {
        difference = 0
    }

    fun samplePowerLV(pl: Int) {
        // pl is the current power level
        difference = pl - powerLevel
        powerLevel = pl
    }
}